src/perf/bin/appmain.cpp

```diff
@@ -0,0 +1,228 @@
+/*++
+
+    Copyright (c) Microsoft Corporation.
+    Licensed under the MIT License.
+
+Abstract:
+
+    QUIC Perf Main execution runner.
+
+--*/
+
+#define QUIC_TEST_APIS 1
+#include "quic_driver_main.h"
+#include "PerfHelpers.h"
+#include <quic_trace.h>
+#ifdef QUIC_CLOG
+#include "appmain.cpp.clog.h"
+#endif
+
+#ifdef _WIN32
+
+//
+// Name of the driver service for quicperf.sys.
+// Must be defined before quic_driver_helpers.h is included
+//
+#define QUIC_DRIVER_NAME   "quicperf"
+
+#include <winioctl.h>
+#include "PerfIoctls.h"
+#include "quic_driver_helpers.h"
+
+#endif
+
+extern "C" _IRQL_requires_max_(PASSIVE_LEVEL) void QuicTraceRundown(void) { }
+
+QUIC_STATUS
+QuicUserMain(
+    _In_ int argc,
+    _In_reads_(argc) _Null_terminated_ char* argv[],
+    _In_ bool KeyboardWait,
+    _In_ PerfSelfSignedConfiguration* SelfSignedConfig
+    ) {
+    QUIC_EVENT StopEvent;
+    QuicEventInitialize(&StopEvent, true, false);
+
+    QUIC_STATUS RetVal = QuicMainStart(argc, argv, StopEvent, SelfSignedConfig);
```

Comment:
Instead of `RetVal` let's use `Status` since that's what we use pretty much throughout the code base.